ui_framework/doc_site/src/views/tool_bar/tool_bar.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+import {
+  KuiToolBar,
+  KuiToolBarSearchBox,
+  KuiButton,
+  KuiButtonIcon,
+  KuiButtonGroup,
+} from '../../../../components';
+
+export const ToolBar = () => (
+  <KuiToolBar>
+    <KuiToolBarSearchBox onFilter={() => {}}/>
```

Comment:
Is it a space because of the closing tag, or should there always be a space between the last `>` or `/>` and the prop before it?  Because it looks like I have quite a few instances of e.g.:
`<div className="kuiToolBarText">`.  Should I change them to e.g.: `<div className="kuiToolBarText" >`?